### ISSUE TITLE
Harmless (?) tracebacks from DataView-related tests

### ISSUE THREAD

**Issue by rahulporuri** [enthought/pyface#625]
- [ ] test_header_visible (pyface.data_view.tests.test_data_view_widget.TestWidget)
- [ ] test_header_visible_before_control (pyface.data_view.tests.test_data_view_widget.TestWidget)
- [ ] test_lifecycle (pyface.data_view.tests.test_data_view_widget.TestWidget)

<details>

```
test_header_visible (pyface.data_view.tests.test_data_view_widget.TestWidget) ... Traceback (most recent call last):

  File "/home/travis/.edm/envs/pyface-test-3.6-wx/lib/python3.6/site-packages/pyface-7.1.0.dev60-py3.6.egg/pyface/ui/wx/data_view/data_view_model.py", line 180, in GetColumnType

    return type_hint_to_variant.get(value_type.type_hint, "string")

AttributeError: 'IntValue' object has no attribute 'type_hint'

Traceback (most recent call last):

  File "/home/travis/.edm/envs/pyface-test-3.6-wx/lib/python3.6/site-packages/pyface-7.1.0.dev60-py3.6.egg/pyface/ui/wx/data_view/data_view_model.py", line 180, in GetColumnType

    return type_hint_to_variant.get(value_type.type_hint, "string")

AttributeError: 'IntValue' object has no attribute 'type_hint'

Traceback (most recent call last):

  File "/home/travis/.edm/envs/pyface-test-3.6-wx/lib/python3.6/site-packages/pyface-7.1.0.dev60-py3.6.egg/pyface/ui/wx/data_view/data_view_model.py", line 180, in GetColumnType

    return type_hint_to_variant.get(value_type.type_hint, "string")

AttributeError: 'IntValue' object has no attribute 'type_hint'

Traceback (most recent call last):

  File "/home/travis/.edm/envs/pyface-test-3.6-wx/lib/python3.6/site-packages/pyface-7.1.0.dev60-py3.6.egg/pyface/ui/wx/data_view/data_view_model.py", line 180, in GetColumnType

    return type_hint_to_variant.get(value_type.type_hint, "string")

AttributeError: 'IntValue' object has no attribute 'type_hint'

Traceback (most recent call last):

  File "/home/travis/.edm/envs/pyface-test-3.6-wx/lib/python3.6/site-packages/pyface-7.1.0.dev60-py3.6.egg/pyface/ui/wx/data_view/data_view_model.py", line 180, in GetColumnType

    return type_hint_to_variant.get(value_type.type_hint, "string")

AttributeError: 'IntValue' object has no attribute 'type_hint'

Traceback (most recent call last):

  File "/home/travis/.edm/envs/pyface-test-3.6-wx/lib/python3.6/site-packages/pyface-7.1.0.dev60-py3.6.egg/pyface/ui/wx/data_view/data_view_model.py", line 180, in GetColumnType

    return type_hint_to_variant.get(value_type.type_hint, "string")

AttributeError: 'IntValue' object has no attribute 'type_hint'

Traceback (most recent call last):

  File "/home/travis/.edm/envs/pyface-test-3.6-wx/lib/python3.6/site-packages/pyface-7.1.0.dev60-py3.6.egg/pyface/ui/wx/data_view/data_view_model.py", line 180, in GetColumnType

    return type_hint_to_variant.get(value_type.type_hint, "string")

AttributeError: 'IntValue' object has no attribute 'type_hint'

ok

test_header_visible_before_control (pyface.data_view.tests.test_data_view_widget.TestWidget) ... Traceback (most recent call last):

  File "/home/travis/.edm/envs/pyface-test-3.6-wx/lib/python3.6/site-packages/pyface-7.1.0.dev60-py3.6.egg/pyface/ui/wx/data_view/data_view_model.py", line 180, in GetColumnType

    return type_hint_to_variant.get(value_type.type_hint, "string")

AttributeError: 'IntValue' object has no attribute 'type_hint'

Traceback (most recent call last):

  File "/home/travis/.edm/envs/pyface-test-3.6-wx/lib/python3.6/site-packages/pyface-7.1.0.dev60-py3.6.egg/pyface/ui/wx/data_view/data_view_model.py", line 180, in GetColumnType

    return type_hint_to_variant.get(value_type.type_hint, "string")

AttributeError: 'IntValue' object has no attribute 'type_hint'

Traceback (most recent call last):

  File "/home/travis/.edm/envs/pyface-test-3.6-wx/lib/python3.6/site-packages/pyface-7.1.0.dev60-py3.6.egg/pyface/ui/wx/data_view/data_view_model.py", line 180, in GetColumnType

    return type_hint_to_variant.get(value_type.type_hint, "string")

AttributeError: 'IntValue' object has no attribute 'type_hint'

Traceback (most recent call last):

  File "/home/travis/.edm/envs/pyface-test-3.6-wx/lib/python3.6/site-packages/pyface-7.1.0.dev60-py3.6.egg/pyface/ui/wx/data_view/data_view_model.py", line 180, in GetColumnType

    return type_hint_to_variant.get(value_type.type_hint, "string")

AttributeError: 'IntValue' object has no attribute 'type_hint'

Traceback (most recent call last):

  File "/home/travis/.edm/envs/pyface-test-3.6-wx/lib/python3.6/site-packages/pyface-7.1.0.dev60-py3.6.egg/pyface/ui/wx/data_view/data_view_model.py", line 180, in GetColumnType

    return type_hint_to_variant.get(value_type.type_hint, "string")

AttributeError: 'IntValue' object has no attribute 'type_hint'

Traceback (most recent call last):

  File "/home/travis/.edm/envs/pyface-test-3.6-wx/lib/python3.6/site-packages/pyface-7.1.0.dev60-py3.6.egg/pyface/ui/wx/data_view/data_view_model.py", line 180, in GetColumnType

    return type_hint_to_variant.get(value_type.type_hint, "string")

AttributeError: 'IntValue' object has no attribute 'type_hint'

Traceback (most recent call last):

  File "/home/travis/.edm/envs/pyface-test-3.6-wx/lib/python3.6/site-packages/pyface-7.1.0.dev60-py3.6.egg/pyface/ui/wx/data_view/data_view_model.py", line 180, in GetColumnType

    return type_hint_to_variant.get(value_type.type_hint, "string")

AttributeError: 'IntValue' object has no attribute 'type_hint'

ok

test_lifecycle (pyface.data_view.tests.test_data_view_widget.TestWidget) ... Traceback (most recent call last):

  File "/home/travis/.edm/envs/pyface-test-3.6-wx/lib/python3.6/site-packages/pyface-7.1.0.dev60-py3.6.egg/pyface/ui/wx/data_view/data_view_model.py", line 180, in GetColumnType

    return type_hint_to_variant.get(value_type.type_hint, "string")

AttributeError: 'IntValue' object has no attribute 'type_hint'

Traceback (most recent call last):

  File "/home/travis/.edm/envs/pyface-test-3.6-wx/lib/python3.6/site-packages/pyface-7.1.0.dev60-py3.6.egg/pyface/ui/wx/data_view/data_view_model.py", line 180, in GetColumnType

    return type_hint_to_variant.get(value_type.type_hint, "string")

AttributeError: 'IntValue' object has no attribute 'type_hint'

Traceback (most recent call last):

  File "/home/travis/.edm/envs/pyface-test-3.6-wx/lib/python3.6/site-packages/pyface-7.1.0.dev60-py3.6.egg/pyface/ui/wx/data_view/data_view_model.py", line 180, in GetColumnType

    return type_hint_to_variant.get(value_type.type_hint, "string")

AttributeError: 'IntValue' object has no attribute 'type_hint'

Traceback (most recent call last):

  File "/home/travis/.edm/envs/pyface-test-3.6-wx/lib/python3.6/site-packages/pyface-7.1.0.dev60-py3.6.egg/pyface/ui/wx/data_view/data_view_model.py", line 180, in GetColumnType

    return type_hint_to_variant.get(value_type.type_hint, "string")

AttributeError: 'IntValue' object has no attribute 'type_hint'

Traceback (most recent call last):

  File "/home/travis/.edm/envs/pyface-test-3.6-wx/lib/python3.6/site-packages/pyface-7.1.0.dev60-py3.6.egg/pyface/ui/wx/data_view/data_view_model.py", line 180, in GetColumnType

    return type_hint_to_variant.get(value_type.type_hint, "string")

AttributeError: 'IntValue' object has no attribute 'type_hint'

Traceback (most recent call last):

  File "/home/travis/.edm/envs/pyface-test-3.6-wx/lib/python3.6/site-packages/pyface-7.1.0.dev60-py3.6.egg/pyface/ui/wx/data_view/data_view_model.py", line 180, in GetColumnType

    return type_hint_to_variant.get(value_type.type_hint, "string")

AttributeError: 'IntValue' object has no attribute 'type_hint'

Traceback (most recent call last):

  File "/home/travis/.edm/envs/pyface-test-3.6-wx/lib/python3.6/site-packages/pyface-7.1.0.dev60-py3.6.egg/pyface/ui/wx/data_view/data_view_model.py", line 180, in GetColumnType

    return type_hint_to_variant.get(value_type.type_hint, "string")

AttributeError: 'IntValue' object has no attribute 'type_hint'

ok
```

</details>

**Comment by corranwebster** [comment]
These are fixed in #574

**Comment by aaronayres35** [comment]
closing since #574 has been merged
